uaclient/cli.py

```diff
@@ -385,8 +385,8 @@ def action_fix(args, *, cfg, **kwargs):
         ).format(args.security_issue)
         raise exceptions.UserFacingError(msg)
 
-    security.fix_security_issue_id(cfg, args.security_issue)
-    return 0
+    fix_status = security.fix_security_issue_id(cfg, args.security_issue)
+    return int(fix_status.value)
```

Comment:
Curious, why is `int() needed?

Reply:
We shouldn't need this actually, I am just ensuring that if for some reason the enum value returns a string, it will be converted. But maybe this is unnecessary. Let me know what you think and I can remove it if needed

Reply:
I don't think the `int()` is necessary but it doesn't hurt either. Up to you, but if you want to try removing it then as long as the integration tests added in this PR still pass, I don't think we need to wait for jenkins again as the current results look good.

Reply:
I have removed the `int()` part